### PR TITLE
merge solver and amr libs

### PR DESCRIPTION
**Pull request by nicolasaunai**

### CODE REVIEW

File: src/amr/multiphysics_integrator.h [hunk 1]
```diff
@@ -18,14 +18,14 @@
 #include "amr/messengers/messenger.h"
 #include "amr/tagging/tagger.h"
 
-#include "solver/physical_models/hybrid_model.h"
-#include "solver/physical_models/mhd_model.h"
-#include "solver/physical_models/physical_model.h"
-#include "solver/solvers/solver.h"
-#include "solver/messenger_registration.h"
-#include "solver/level_initializer/level_initializer.h"
+#include "physical_models/hybrid_model.h"
```
Comment (PhilipDeegan) [inline src/amr/multiphysics_integrator.h:21]: this compiles, but it probably shouldn't.

ie. there's a bunch of extra "-Isrc/$module" that are in cmake but I don't think should be

all includes should be from src

File: src/amr/multiphysics_integrator.h [hunk 2]
```diff
@@ -18,14 +18,14 @@
...
+#include "physical_models/mhd_model.h"
+#include "physical_models/physical_model.h"
+#include "solvers/solver.h"
+#include "messenger_registration.h"
+#include "level_initializer/level_initializer.h"
```
Comment (PhilipDeegan) [inline src/amr/multiphysics_integrator.h:26]: this compiles, but it probably shouldn't.

ie. there's a bunch of extra "-Isrc/$module" that are in cmake but I don't think should be

all includes should be from src

**Review by PhilipDeegan — APPROVED** [review]
just the includes otherwise LGTM

File: src/amr/multiphysics_integrator.h (hunk 2 above)
Comment (nicolasaunai) [reply src/amr/multiphysics_integrator.h:26]: done

**Pull request merged**